src/unity/python/turicreate/visualization/show.py

```diff
@@ -7,6 +7,17 @@
 from __future__ import division as _
 from __future__ import absolute_import as _
 
+def get_client_app_path():
+    import sys
+    import os
+    (tcviz_dir, _) = os.path.split(os.path.dirname(__file__))
+
+    if sys.platform == 'darwin':
+        return os.path.join(tcviz_dir, 'Turi Create Visualization.app', 'Contents', 'MacOS', 'Turi Create Visualization')
+
+    if sys.platform == 'linux2':
+        return os.path.join(tcviz_dir, 'TcViz', 'visualization_client')
```

Comment:
Let's keep the names consistent across platforms when possible - use "Turi Create Visualization" for both the directory name and the binary filename on Linux.